docs/vote-escrow/realms.mdx

```diff
@@ -0,0 +1,164 @@
+---
+id: realms
+title: Staking veHNT on Realms
+pagination_label: Realms
+sidebar_label: Staking veHNT on Realms
+description: Helium on Realms Documentation
+image: https://docs.helium.com/img/link-image.png
+slug: /vote-escrow/realms
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+<img className="docsheader" src={useBaseUrl('/img/blockchain/realmsheader.png')} />
+
+Like many other Solana Ecosystem projects, Helium utilizes [Realms](https://realms.today) to organize and manage vote escrow tokens, delegation, and more.
+Stakes can be created and later managed within Realms at any time.
+
+## Landrush
+For the first 10 days following the Solana migration, HNT holders are elegible to stake their HNT for a 'landrush' bonus of 3x the lock amount.<sup>(</sup>[^1]<sup>,</sup>[^2]<sup>)</sup>
+
+If a Landrush stake is moved after the first 10 days have lapsed, the landrush bonus is forfeited.
```

Comment:
If you transfer out of or split tokens out of a Landrush position the bonus is forfeited.